internal/services/applications/parse/fallback_public_client.go

```diff
@@ -69,3 +69,13 @@ func (id FallbackPublicClientId) Segments() []resourceids.Segment {
 func (id FallbackPublicClientId) String() string {
 	return fmt.Sprintf("Fallback Public Client (Application ID: %q)", id.ApplicationId)
 }
+
+func (id FallbackPublicClientId) FromParseResult(input resourceids.ParseResult) error {
```

Comment:
This'll need to be a pointer:

```suggestion
func (id *FallbackPublicClientId) FromParseResult(input resourceids.ParseResult) error {
```